### PR TITLE
Feature/QA Finds

### PR DESCRIPTION
**Pull request by pattisdr**
- [x] Add help text of why study comments are disabled
- [x] Exit url, short description, purpose help text
- [x] If logging out experimenter side, redirect back to oauth login page instead of Lookit-side login page
- [x] Don't show login link on oauth login page
- [x] Label Lookit login page as "Participant Login" 
- [x] Show navbar on dashboard page, with "Experimenter" text not a link if user hasn't been added to any organization groups
- [x] Add help text on Researcher detail page for users with no organization groups
- [x] Open OSF profile in new tab since it takes you outside the app
- [x] Oauth page says "Login to Experimenter" 
- [x] Fix issue with logging in oauth without first or last name filled in.
- [x] help text in confirmation modal if intending to archive or deactivate a study

### CODE REVIEW

File: accounts/templates/accounts/researcher_detail.html [hunk 1]
```diff
@@ -130,7 +131,11 @@
                 {% if perms.accounts.can_edit_organization  %}
                     <a href="#" id="user_permissions" data-type="select" data-url="{% url 'exp:researcher-detail' user.id %}" data-params="{csrfmiddlewaretoken:'{{csrf_token}}'}" data-pk="{{ user.id }}" data-url="/post" data-title="Select permissions">
                         {{ user.display_permission }}
+
                     </a>
+                    {% if user.display_permission == 'No organization groups' %}
```
Comment (cwisecarver) [inline accounts/templates/accounts/researcher_detail.html]: Nice. That's much better.

File: exp/templates/exp/_navigation.html [hunk 2]
```diff
@@ -7,7 +7,12 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="{% url 'exp:study-list' %}">Experimenter</a>
+      {% if request.user.groups.exists %}
```
Comment (cwisecarver) [inline exp/templates/exp/_navigation.html]: You could also check `request.user.is_researcher` and then send them to `exp:dashboard` that way if they wanted to check their status and they hadn't been approved yet they could still get the purgatory page telling them to wait.

File: exp/templates/exp/dashboard.html [hunk 3]
```diff
@@ -5,16 +5,22 @@
...
+    {% bootstrap_messages %}
   <div class="panel panel-default">
     <div class="panel-heading">
       <h1>{% trans "Dashboard" %}</h1>
     </div>
     <div class="panel-body">
-      <p>You've successfully signed into Experimenter. An administrator will be contacted to grant you permissions. You will receive a notification by email when this is completed.</p>
+      <p>You've successfully signed into Experimenter! </p>
+      <p>
+          An administrator will be contacted to grant you permissions. You will receive a notification by email when this is completed.
```
Comment (cwisecarver) [inline exp/templates/exp/dashboard.html]: We should probably setup this so-called notification by email business... Maybe a post save hook on users? If they didn't have groups and now they have groups then use django's email framework (examples are in the workflow callbacks on studies) to send an email which Nici should hopefully have a template for. I'll make a ticket.

**Comment by pattisdr** [comment]
@cwisecarver this is ready for review, I will stop pushing commits here and switch to another branch.

File: exp/templates/exp/dashboard.html (hunk 3 above)
Comment (cwisecarver) [inline exp/templates/exp/dashboard.html]: https://openscience.atlassian.net/projects/LEI/issues/LEI-439?filter=allopenissues

**Pull request merged**